release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -0,0 +1,573 @@
+=== Deprecated features
+
+[id="ocp-4-9-removed-features"]
+=== Removed features
+
+[id="ocp-4-9-bug-fixes"]
+== Bug fixes
+
+*assisted-installer*
+
+*Bare Metal Hardware Provisioning*
+
+*Builds*
+
+*Cloud Compute*
+
+*Cloud Credential Operator*
+
+*Cluster Version Operator*
+
+*Compliance Operator*
+
+*Console Kubevirt Plug-in*
+
+*Console Storage Plug-in*
+
+*DNS*
+
+*etcd*
+
+*Image Registry*
+
+*ImageStreams*
+
+*Insights Operator*
+
+*Installer*
+
+*kube-apiserver*
+
+*Machine Config Operator*
+
+*Metering Operator*
+//TODO: remove if this Operator is removed as planned
+
+*Monitoring*
+
+*Networking*
+
+*Node*
+
+*oauth-apiserver*
+
+*oc*
+
+*Operator Lifecycle Manager (OLM)*
+
+*Operator SDK*
+
+*openshift-apiserver*
+
+*Performance Addon Operator*
+
+*RHCOS*
+
+*Routing*
+
+*Samples*
+
+*service-ca*
+
+*Storage*
+
+*Web console (Administrator perspective)*
+
+*Web console (Developer perspective)*
+
+*Windows Containers*
+
+[id="ocp-4-9-technology-preview"]
+== Technology Preview features
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+* *TP*: _Technology Preview_
+* *GA*: _General Availability_
+* *-*: _Not Available_
+* *DEP*: _Deprecated_
+
+//TODO: remove anything that has been GA since 4.6
+
+.Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |OCP 4.7 |OCP 4.8 |OCP 4.9
+
+|Precision Time Protocol (PTP)
+|TP
+|TP
+|
+
+|`oc` CLI plug-ins
+|TP
+|GA
+|
+
+|Descheduler
+|GA
+|GA
+|
+
+|HPA for memory utilization
+|GA
+|GA
+|
+
+|Service Binding
+|TP
+|TP
+|
+
+|Raw Block with Cinder
+|TP
+|GA
+|
+
+|CSI volume snapshots
+|GA
+|GA
+|
+
+|CSI volume expansion
+|TP
+|TP
+|
+
+|vSphere Problem Detector Operator
+|GA
+|GA
+|
+
+|CSI Azure Disk Driver Operator
+|-
+|TP
+|
+
+|CSI GCP PD Driver Operator
+|TP
+|GA
+|
+
+|CSI OpenStack Cinder Driver Operator
+|TP
+|TP
+|
+
+|CSI AWS EBS Driver Operator
+|TP
+|TP
+|
+
+|CSI automatic migration
+|-
+|TP
+|
+
+|CSI inline ephemeral volumes
+|TP
+|TP
+|
+
+|CSI vSphere Driver Operator
+|-
+|TP
+|
+
+|Automatic device discovery and provisioning with Local Storage Operator
+|TP
+|TP
+|
+
+|OpenShift Pipelines
+|TP
+|GA
+|
+
+|OpenShift GitOps
+|TP
+|GA
+|
+
+|OpenShift sandboxed containers
+|-
+|TP
+|
+
+|Vertical Pod Autoscaler
+|TP
+|GA
+|
+
+|Cron jobs
+|TP
+|GA
+|
+
+|PodDisruptionBudget
+|TP
+|GA
+|
+
+|Adding kernel modules to nodes with kvc
+|TP
+|TP
+|
+
+|Egress router CNI plug-in
+|TP
+|GA
+|
+
+|Scheduler profiles
+|TP
+|TP
+|
+
+|Non-preempting priority classes
+|TP
+|TP
+|
+
+|Kubernetes NMState Operator
+|TP
+|TP
+|
+
+|Assisted Installer
+|TP
+|TP
+|
+
+|AWS Security Token Service (STS)
+|TP
+|GA
+|GA
```

Comment:
This is the only one that has something in the 4.9 column, just checking.

Reply:
the only one I know about :wink: